### PR TITLE
add products to category, adjust unit-tests

### PR DESCRIPTION
**Pull request by DerCalli**
**Feature**

Add products to data-layer inside category within unit-tests

For plugins located at:
https://github.com/fond-of/spryker-google-tag-manager-category-product-connector-

### CODE REVIEW

**Review by SpiderDan98 — APPROVED** [review]

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    /**
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    protected function addGoogleTagManagerCategoryProductPlugins(Container $container): Container
+    {
+        $container->set(static::PLUGINS_GOOGLE_TAG_MANAGER_CATEGORY_PRODUCT, function () {
```
Comment (julianhyatt) [inline src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php:31]: static function

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/Model/GoogleTagManagerCategoryModel.php [hunk 2]
```diff
@@ -77,4 +90,35 @@ public function getCategorySize(array $params): array
             ModuleConstants::FIELD_CATEGORY_SIZE => count($params[ModuleConstants::PARAM_PRODUCTS]),
         ];
     }
+
+    /**
+     * @param array|string $params
```
Comment (julianhyatt) [inline src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/Model/GoogleTagManagerCategoryModel.php]: page is missing

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+namespace FondOfSpryker\Yves\GoogleTagManagerCategoryConnector;
+
+use Spryker\Yves\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Yves\Kernel\Container;
+
+class GoogleTagManagerCategoryConnectorDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const PLUGINS_GOOGLE_TAG_MANAGER_CATEGORY_PRODUCT = 'PLUGINS_GOOGLE_TAG_MANAGER_CATEGORY_PRODUCT';
+
+    public function provideDependencies(Container $container): Container
```
Comment (julianhyatt) [inline src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php:17]: missing

**Review by julianhyatt — COMMENTED** [review]
you should run phpcs

File: src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php (hunk 3 above)
Comment (DerCalli) [reply src/FondOfSpryker/Yves/GoogleTagManagerCategoryConnector/GoogleTagManagerCategoryConnectorDependencyProvider.php:17]: thx!

strange, i already run phpcbf. No fixable errors found but doc-block is still missing.

**Pull request merged**